docs/conf.py

```diff
@@ -89,9 +89,18 @@
     ],
     # Configure secondary (right) side bar
     "show_toc_level": 3,                     # Show all subsections of notebooks
+    "show_nav_level": 2,
     "secondary_sidebar_items": ["page-toc"]  # Omit 'show source' link that that shows notebook in json format
 }
 
+html_sidebars = {
+    "community/index": [
+        "sidebar-nav-bs",
+    ],
```

Comment:
```suggestion
```
this is redundant

Reply:
Not sure is we need any of those, maybe lets wait for #59 here as well?

Reply:
I removed those and the page still looks exactly the same...